remote/gogs/helper.go

```diff
@@ -66,13 +66,29 @@ func buildFromPush(hook *PushHook) *model.Build {
 		hook.Repo.Url,
 		fixMalformedAvatar(hook.Sender.Avatar),
 	)
+
+	eventType := "Undefined"
+	message := "No message"
+
+	switch {
+	case hook.RefType == "tag":
+		eventType = model.EventTag
+		message = "Create Tag " + hook.Ref
```

Comment:
For a tag creation there is no commit message in the payload, here is the example posted in issue #1412 : 

```
{
  "secret": "hidden",
  "ref": "v1.2.6",
  "ref_type": "tag",
  "repository": {
    "id": 1,
    "name": "repository",
    "url": "https://gogs.company.com/project/repository",
    "ssh_url": "git@gogs.company.com:project/repository.git",
    "clone_url": "https://gogs.company.com/project/repository.git",
    "description": "",
    "website": "",
    "watchers": 13,
    "owner": {
      "name": "project",
      "email": "",
      "username": "project"
    },
    "private": true
  },
  "sender": {
    "login": "jose.moreira",
    "id": 1,
    "avatar_url": "https://gogs.company.com//avatar/6b2f1bb07eb368151c88af45764967df"
  }

```}